builtin/logical/consul/path_roles.go

```diff
@@ -71,6 +71,20 @@ Defaults to 'client'.`,
 				Description: "Use ttl instead.",
 				Deprecated:  true,
 			},
+
+			"consul_namespace": {
+				Type:    framework.TypeString,
+				Default: "default",
```

Comment:
How is `"default"` used? Is it a special term that consul understands, different from `""`?

Reply:
I believe that default is the literal name of the default namespace and partition both, and so consul does understand it. However, removing the default definition here still results in using the "default" in consul, so perhaps it is unnecessary.

Reply:
I removed it because it is redundant https://github.com/hashicorp/vault/pull/13850/commits/45a4d65bcbbef3d2adb5892ed048f5fbf262ad72